product_multi_image/README.rst

```diff
@@ -0,0 +1,39 @@
+Multiple images in products
+===========================
+
+This module implements the possibility to have multiple images for a product,
+that it's to say, an image gallery.
+
```

Comment:
This is missing some important parts such as usage instructions, try on runbot, report issues, etc.